docusaurus/docs/create-a-plugin/develop-a-plugin/build-a-data-source-backend-plugin.md

```diff
@@ -175,19 +154,21 @@ Implementing authentication allows your plugin to access protected resources lik
      ...
    }
    ```
-1. Rebuild frontend parts of the plugin to _dist_ directory:
-
-```bash
-yarn build
-```
 
 1. Restart your Grafana instance.
 1. Open Grafana in your web browser.
+1. Verify that alerting is now supported by navigating to your created data source. You should an "Alerting supported" message in the Settings view.
```

Comment:
This is fun since its changed between versions 9 and 10 as well, and we currently use `9.5.3` by default in create-plugin. 

We may need to make this simpler/generic, advise which version these instructions apply for, and link to the docs for alerting.